pom.xml

```diff
@@ -91,5 +92,11 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest</artifactId>
+            <version>${hamcrest.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
What's the reason for this? I don't see any `org.hamcrest` classes used.

Reply:
We are using for assertions like `assertThat`, and matchers like `is`.
I did not expect that `org.junit.jupter` comes with assertions, maybe should be better to use them instead of using `org.hamrest`?

Reply:
No sorry, my mistake. You were using it and we use it even in other Strimzi projects so it's fine with me.